coursier.bzl

```diff
@@ -552,15 +556,25 @@ def _coursier_fetch_impl(repository_ctx):
         mirror_urls = [url + "/" + primary_artifact_path for url in repository_urls]
         artifact.update({"mirror_urls": mirror_urls})
 
-        # Compute the sha256 of the file.
-        hasher_command = _generate_java_jar_command(
-            repository_ctx, repository_ctx.path(repository_ctx.attr._sha256_hasher))
-        exec_result = repository_ctx.execute(hasher_command + [repository_ctx.path(artifact["file"])])
-        if exec_result.return_code != 0:
-            fail("Error while obtaining the sha256 checksum of "
-                    + artifact["file"] + ": " + exec_result.stderr)
-        # Update the SHA-256 checksum in-place.
-        artifact.update({"sha256": exec_result.stdout})
+        hasher_command.append(repository_ctx.path(artifact["file"]))
+
+    exec_result = repository_ctx.execute(hasher_command)
+    if exec_result.return_code != 0:
+        fail("Error while obtaining the sha256 checksums: " + exec_result.stderr)
+
+    shas = {}
+    for line in exec_result.stdout.splitlines():
+      parts = line.split(" ")
+      path = repository_ctx.path(parts[1])
+      shas[path] = parts[0]
```

Comment:
use `artifact["coord"]` as the key? that should solve the CI failure.

Reply:
I didn't realize path wasn't hashable on some versions. How do you suggest going about that for coord? Pass a series of concatenations of file and coord to the java process like `coord:file coord:file`, then return `coord:sha coord:sha`?

Reply:
Hmm, you could also cast path to `str()`. Should be a fairly fast opertation.

Reply:
Ok I'll try that first